### ISSUE TITLE
List verification failure

### ISSUE THREAD

**Issue by miraslau** [oleksiyp/mockk#10]
In the last line it will fail, but it shouldn't, as much I understand:
```
@Test fun list() {
        val f: (List<Int>) -> Unit = mockk {
            every { this@mockk.invoke(any()) } just Runs
        }
        val list = listOf(1, 2, 3, 4, 5)
        val listClone = list.toList()
        f(list)
        Assert.assertEquals(list, listClone)
        verify { f(listClone) }
}
```

**Comment by oleksiyp** [comment]
Will check

**Comment by oleksiyp** [comment]
Thank you for reporting this bug. I fixed it in 1.5.5 release. Please check if it works for you and close the ticket. Thanks

**Comment by oleksiyp** [comment]
Ok, will check that as well. Can you tell if first case was resolved?

**Comment by miraslau** [comment]
Yeah, with integers it works well, the same as with real objects instead of mock, as I notice in my previous post.

**Comment by oleksiyp** [comment]
Fixed in 1.5.6 version. The problem with codebase is that it was written during just one week. I wanted to finish MVP. So tests really sucks. My plan is following: I have API separated from implementation, so I would just rewrite implementation in TDD approach.

**Comment by miraslau** [comment]
The problem is gone. Thank you.
